Changelog.md

```diff
@@ -1,3 +1,9 @@
+## 4.4.2 - Unreleased
+
+### Fixed
+
+-   Proxy settings are passed on to nRF Util CLI
```

Comment:
To be honest, this phrasing does not tell me what the issue was. Should I consider passing the settings an issue or lack of passing them an issue?
Something like this would be clearer:
```suggestion
-   Passing of proxy settings to nRF Util CLI. They are now passed on automatically.
```
or 
```suggestion
-   Proxy settings are now correctly passed on to nRF Util CLI.
```